src/teams.mts

```diff
@@ -0,0 +1,57 @@
+import { userInfo, homedir } from 'os';
+import { readFileSync } from 'fs';
+import LoggerFactory from './logger.mjs';
+
+const logger = LoggerFactory();
+const teamsUrl = 'https://team.rei-cloud.com/rs/teams';
+
+interface Team {
+  id: string,
+  members: {
+    name: string,
+    userId: string,
+  }[]
+}
+
+const getTeam = (
+  userId: string,
+  teamsArray: Team[],
+) => teamsArray
+  .find(
+    (team: Team) => team.members
+      .find(
+        (member) => member.userId === userId,
+      ),
+  )?.id;
+
+/* c8 ignore start */
+const getHardshellToken = () => readFileSync(`${homedir()}/hardshell.jwt`, 'utf-8');
```

Comment:
Any reason we're using `readFileSync` instead of `readFile`?

```typescript
import { readFile } from 'node:fs'
```

Reply:
Updated